client/types/index.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module "react-twitter-embed" {
```

Comment:
The "react-twitter-embed" package didnt have a @types package, so had to write my own

Reply:
The extra mile. Great.